android-agent/src/main/kotlin/io/opentelemetry/android/agent/OtelRumConfigExtensions.kt

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package io.opentelemetry.android.agent
+
+import io.opentelemetry.android.config.OtelRumConfig
+import io.opentelemetry.android.instrumentation.AndroidInstrumentationRegistry
+import io.opentelemetry.android.instrumentation.activity.ActivityLifecycleInstrumentation
+import io.opentelemetry.android.instrumentation.anr.AnrInstrumentation
+import io.opentelemetry.android.instrumentation.common.ScreenNameExtractor
+import io.opentelemetry.android.instrumentation.crash.CrashDetails
+import io.opentelemetry.android.instrumentation.crash.CrashReporterInstrumentation
+import io.opentelemetry.android.instrumentation.fragment.FragmentLifecycleInstrumentation
+import io.opentelemetry.android.instrumentation.network.NetworkChangeInstrumentation
+import io.opentelemetry.android.instrumentation.slowrendering.SlowRenderingInstrumentation
+import io.opentelemetry.android.internal.services.network.data.CurrentNetwork
+import io.opentelemetry.api.trace.Tracer
+import io.opentelemetry.instrumentation.api.instrumenter.AttributesExtractor
+import java.time.Duration
+
+/**
```

Comment:
This change isn't really needed, I added it because it seemed handy to have those options in `OtelRumConfig`.

Reply:
Oh yeah interesting, I hadn't thought of doing it with extension methods.... 🤔